cli/tools/registry/pm.rs

```diff
@@ -25,22 +28,149 @@ use crate::file_fetcher::FileFetcher;
 use crate::jsr::JsrFetchResolver;
 use crate::npm::NpmFetchResolver;
 
+enum ConfigFile {
```

Comment:
The term `ConfigFile` usually refers to the deno.json config file in the codebase. It would be nice to give this a distinct name even though it's a private enum (ex. `DenoOrPackageJson`).